<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditableScriptVariableTableModelEditDialog</class>
 <widget class="QDialog" name="EditableScriptVariableTableModelEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Script Variable Editing</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item row="5" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Mandatory:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QComboBox" name="comboBoxTypes">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Boolean</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Int32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Int64</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Double</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>String</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Binary</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Direction:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="EditableSubtypeListWidget" name="widgetVariableSubtypesEditor" native="true">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight|Qt::AlignTop">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QComboBox" name="comboBoxDirection">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Input</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Output</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bidirectional</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPlainTextEdit" name="plainTextEditDescription">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEditName">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignRight|Qt::AlignVCenter">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QCheckBox" name="checkBoxMandatory">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditableSubtypeListWidget</class>
   <extends>QWidget</extends>
   <header>data/editor/common/EditableSubtypeListWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
